### DISCUSSION TITLE
Purchased licence but videos corrupted / unwatchable

### DISCUSSION THREAD

**Discussion by Dark-Obsidian** [aclap-dev/video-downloadhelper#413] [License & Purchase requests]
### What's your issue?

I purchased a licence ~1 hour ago and downloaded 3-4 videos, however when I tried to watch them now they are all corrupted and unwatchable.

### The 10 first digits (only 10) of your license number or email address

8d51cfd567

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
We need your versions: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
And a link for a public video with this issue to test.
